langwatch/src/pages/settings/triggers.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  HStack,
+  Heading,
+  LinkBox,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Spacer,
+  Table,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  VStack,
+  useToast,
+} from "@chakra-ui/react";
+import { MoreVertical } from "react-feather";
+import SettingsLayout from "../../components/SettingsLayout";
+import { useOrganizationTeamProject } from "../../hooks/useOrganizationTeamProject";
+import { api } from "../../utils/api";
+
+import { DeleteIcon } from "@chakra-ui/icons";
+import { Switch } from "@chakra-ui/react";
+
+export default function Members() {
+  const { project } = useOrganizationTeamProject();
+  const toast = useToast();
+
+  const triggers = api.trigger.getTriggers.useQuery({
+    projectId: project?.id ?? "",
+  });
+
+  const toggleTrigger = api.trigger.toggleTrigger.useMutation();
+  const deleteTriggerMutation = api.trigger.deleteById.useMutation();
+
+  const handleToggleTrigger = (triggerId: string, active: boolean) => {
+    toggleTrigger.mutate(
+      { triggerId, active, projectId: project?.id ?? "" },
+      {
+        onSuccess: () => {
+          void triggers.refetch();
+        },
+        onError: (error) => {
+          toast({
+            title: "Failed to update trigger",
+            status: "error",
+            description: error.message,
+            duration: 6000,
+            isClosable: true,
+          });
+        },
+      }
+    );
+  };
+
+  const deleteTrigger = (triggerId: string) => {
+    deleteTriggerMutation.mutate(
+      { triggerId, projectId: project?.id ?? "" },
+      {
+        onSuccess: () => {
+          void triggers.refetch();
```

Comment:
maybe a "Trigger deleted" success message as well? or doesn't make sense UI-wise you think?